Compiler/Globals.py

```diff
@@ -13,6 +13,17 @@
 functions = dict()  # Global dictionary of function_name --> FunctionCompiler objects
 
 
+class LibraryFunctionCompiler:
```

Comment:
I think this class belongs in the same file as the other function compiler, and not in Globals

Reply:
My reasoning to this is because this is the only file where `LibraryFunctionCompiler` is used.